### DISCUSSION TITLE
rustc/clippy hints

### DISCUSSION THREAD

**Discussion by apekros** [zed-industries/zed#10463] [Feature Requests]
I feel like I am missing something, is there a key bind / action in the editor to have the suggestion from rustc/clippy applied? i.e add missing use?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by larseggert** [reply]
Sorry, not going to use Discord.
